install-deps.sh

```diff
@@ -353,6 +353,9 @@ else
 	if $with_seastar; then
 	    build_profiles+=",pkg.ceph.crimson"
 	fi
+ 	if $with_jaeger; then
+	    build_profiles+=",pkg.ceph.jaeger"
+	fi
```

Comment:
could you please reconsider the comment of https://github.com/ceph/ceph/pull/38783/commits/e277cf3ee29063311437d372d1f54bf805a7341e#r664177718

Reply:
added `debian, install-deps.sh` in title: https://github.com/ceph/ceph/pull/38783/commits/fcc70eef230ed7f37c72310edab288d98d8e28f5